src/main/java/uk/ac/ebi/embl/api/validation/helper/Utils.java

```diff
@@ -1283,4 +1283,12 @@ public static AnalysisType getAnalysisType(ValidationScope scope)
 			  return null;
 	  }
   }
+
+	public static StringBuilder getExceptionCause(Throwable t , StringBuilder msg) {
+		while(t != null) {
+			msg.append(t); msg.append("\n");
+			t = t.getCause();
+		}
+		return msg;
+	}
```

Comment:
If you want to get a stack trace this is not the way to do it. This is the canonical way:

{code}
public static String getStackTrace(Throwable ex) {
    StringWriter str = new StringWriter();
    ex.printStackTrace(new PrintWriter(str));
    return str.toString();
  }
{code}